nv_peer_mem.c

```diff
@@ -39,18 +39,67 @@
 #include <linux/export.h>
 #include <linux/hugetlb.h>
 #include <linux/atomic.h>
+#include <linux/pci.h>
 
 
 #include "nv-p2p.h"
 #include <rdma/peer_mem.h>
 
 
 #define DRV_NAME	"nv_mem"
-#define DRV_VERSION	"1.0"
+#define DRV_VERSION	"1.1"
 #define DRV_RELDATE	__DATE__
 
-#define peer_err(FMT, ARGS...) printk(KERN_ERR   DRV_NAME " %s:%d " FMT, __FUNCTION__, __LINE__, ## ARGS)
+#define peer_err(FMT, ARGS...) printk(KERN_ERR   DRV_NAME " %s:%d " FMT, __func__, __LINE__, ## ARGS)
 
+#define INFO_ENABLE 1
+#define peer_info(FMT, ARGS...)		\
```

Comment:
This is not used by the patch, right? Also, if you want programmable prints, I recommend using [dynamic-debug](https://www.kernel.org/doc/Documentation/dynamic-debug-howto.txt) if it is compiled in the kernel.